firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -68,6 +68,8 @@
   private final ExecutorService backgroundExecutor;
   private final ExecutorService networkExecutor;
 
+  private String cachedFid = null;
```

Comment:
please add a comment here, example:
/** FID of this FirebaseInstallations instance. Cached after the first successful read from local storage. */